apps/fabric-examples/src/AppDefinition.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { App as AppBase, IAppDefinition, ExampleStatus } from '@uifabric/example-app-base';
```

Comment:
@micahgodbolt yeah just did it while i was there. I beileve it's unnecessary and confusing, but didn't really want to split out a separate pr.